docs/troubleshooting/password-reset.rst

```diff
@@ -13,11 +13,11 @@ Procedure
 
    sudo systemctl stop scylla-server
 
-| 2. Remove your tables under ``/var/lib/scylla/data/system_auth/``.
+| 2. Remove your tables under ``/var/lib/scylla/data/system_auth_v2/``.
```

Comment:
@kbr-scylla will this work when raft is involved or we need to do something more?